build-tools/automation/azure-pipelines-release-trigger.yaml

```diff
@@ -0,0 +1,45 @@
+# Xamarin.Android Release Definition Trigger
+
+name: $(Build.SourceBranchName)-$(Build.SourceVersion)-$(Rev:r)
+
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: xamarin-android
+    source: Xamarin.Android
+    trigger:
+      stages:
+      - post_build
+
+jobs:
+- job: release_trigger
+  displayName: Upload Info from Triggering Build
+  timeoutInMinutes: 30
+  pool: VSEngSS-MicroBuild2022-1ES
+  workspace:
+    clean: all
+  steps:
+  - checkout: none
+
+  - powershell: |
+      $json = @"

```

Comment:
Recommend using a dictionary instead and convert to json such as
```
$triggeringBuild = @{
    "triggeringBuild" = @{
        "id" = 5
    }
}

$json = $triggeringBuild | ConvertTo-Json -Depth 5
```

Reply:
Plus assuming `Build.TriggeredBy.BuildId` is an int then also recommend leaving the quotes off as in above example

Reply:
Applied, and left the quotes to be safe.